fbw-common/src/systems/shared/src/FbwAircraftSentryClient.ts

```diff
@@ -216,10 +213,10 @@ export class FbwAircraftSentryClient {
       );
     }
 
-    const integrations: Integration[] = [new CaptureConsoleIntegration({ levels: ['error'] })];
+    const integrations = [Sentry.captureConsoleIntegration({ levels: ['error'] })];
```

Comment:
Integrations were converted from classes to functions. Other than that, neither this, nor the browserTracingIntegration changed functionality-wise.